examples/basic/main.tf

```diff
@@ -1,5 +1,5 @@
 ##############################################################################
-# Resource group
+# Use Existing Resource group or Create new one if not exist
```

Comment:
```suggestion
# Use existing resource group or create one if it doesn't exist
```

I generally don't check code comments, but yes, we use sentence case. Also my pet peeve is "create a new..." because you can't create an old anything. (Yes, I do see that's already in line 8)